packages/@react-spectrum/list/stories/ListView.stories.tsx

```diff
@@ -255,6 +293,16 @@ storiesOf('ListView', module)
         <DragExample dragHookOptions={{onDragStart: action('dragStart'), onDragEnd: action('dragEnd')}} />
       </Flex>
     ), {description: {data: 'Folders are non-draggable.'}}
+  )
+  .add(
+    'thumbnails',
```

Comment:
might be nice to have a story with all available slots filled (draggable, icons, action buttons, description, checkboxes, etc) just for testing

Reply:
probably better in a chromatic story?

Reply:
Still seems useful as a story IMO, unless thats a setup design would explicitly recommend against

Reply:
There are some states that are hard to reproduce in a Chromatic story (specifically press, hover, keyboard focus) without doing some class forcing so IMO it would be good to have at one complex story at least for manual verification. Definitely should have a chromatic story still, I can add the thumbnail to my chromatic PR when this goes in